js/releases/index.js

```diff
@@ -199,7 +199,10 @@ class Releases {
 
       return uploadPaths.map((path) =>
         // `execute()` is async and thus we're returning a promise here
-        this.execute(helper.prepareCommand([...args, path], SOURCEMAPS_SCHEMA, newOptions), true)
```

Comment:
so previously, we completely ignored the `live` argument?

Reply:
Not really -- `options` was not supposed to include the `live` key. However, it is (for better or worse) simply typed as `object` so it allows for any key to be passed in. I used this as an easy way to allow anyone calling `uploadSourceMaps(release, options)` to change the `live` value if they want to. We'll use this in the bundler plugin to run `uploadSourceMaps` with `live: rejectOnError`.